apps/rush-lib/assets/rush-init/rush.json

```diff
@@ -270,6 +270,18 @@
    */
   /*[LINE "HYPOTHETICAL"]*/ "hotfixChangeEnabled": false,
 
+  /**
+   * If a warning message includes any substrings listed here, it should be treated as a standard
```

Comment:
I think these should be regular expressions instead of simple substrings.

Reply:
Also the regex's should be anchored to match the whole "message".  But see my note below...